### ISSUE TITLE
Select Browser to Open Link In

### ISSUE THREAD

**Issue by mmcguill** [mmcguill/Strongbox#68]
Allow user to select which browser to open URL in.

**Comment by F3000** [comment]
Idea : Add a webview into the app and open a webpage inline

**Comment by dan-el** [comment]
Why an inline browser? We all have our favorite default browsers and such. What benefit would in inline browser bring compared to a regular mobile browser?

**Comment by F3000** [comment]
You don’t have to switch apps

**Comment by Slummi** [comment]
I don't like the idea of an inline browser. I think it's much better to be able to select your preferred browser to open URLs.
Strongbox is a (very good) password manager based on the KeePass and Password Safe format and it should remain primarily a password manager and not a browser, email client, file manager or anything else. It's in the nature of things that you can't use a password manager standalone.

**Comment by F3000** [comment]
That’s true . A seething to choose between all possibilities should be the best

**Comment by r2evans** [comment]
> Allow user to select which browser to open URL in.

I think this should default to the "default app" for the OS, with the option to select another app. I don't know if it's easy enough in macos/ios to know what apps are browsers, but while that's a good start, it might be that an app (that accepts URLs) might not be recognized by the system, so it may also be useful to allow "Choose application ..." and the command-line required for it.

This issue has been dormant for several years, @strongbox-mark is there anything that is needed (besides time and votes)? Thanks for strongbox!

**Comment by strongbox-mark** [comment]
Yes, all that we need is time and votes :)
